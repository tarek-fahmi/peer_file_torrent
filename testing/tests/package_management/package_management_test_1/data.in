Adding Packages
./testing/bin/btide ./testing/resources/configs/config.cfg
ADDPACKAGE valid_1.bpkg
PACKAGES
QUIT