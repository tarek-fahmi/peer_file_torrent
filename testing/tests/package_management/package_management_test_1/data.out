1. 00bdf3ecd168f7f918c0b37f7e7915f1, part_2/resources/packages/valid_1.data : COMPLETE